pkgs/tools/security/cve-bin-tool/default.nix

```diff
@@ -47,6 +48,7 @@ buildPythonApplication rec {
     pytest-xdist
     pytest-cov
     pytest-asyncio
+    pytest-mock
```

Comment:
Once https://github.com/intel/cve-bin-tool/pull/1665 is merged we can move the testing libs to `checkInputs` and not have them stick around when installed

I guess we could pull it in as a patch but I see no reason to rush

Reply:
We should fix that right now to not draw in test programs at runtime. We also do not need to wait for any upstream PR, just patch it with postPatch.

Reply:
Yeah I guess since we're not running any tests just deleting all lines matching `pytest` will do for now.

maybe `sed '/^pytest/d' -i requirements.txt`

There is also a test file that checks requirements.txt is inline with requirements.csv but I dont think we run that either